.github/workflows/mac-test.yml

```diff
@@ -0,0 +1,34 @@
+name: Mac Build Test CI
+
+on:
+  push:
+    branches:
+      - main
+      - release/**
+    paths-ignore:
+      - README.md
+    tags:
+      - "v*.*.*"
+  pull_request:
+    paths-ignore:
+      - README.md
+
+jobs:
+  mac-test:
+    runs-on: macos-11
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          submodules: recursive
+
+      - name: Add target x86_64-apple-darwin
+        run: rustup target add x86_64-apple-darwin
+
+      - name: Build play-cpp-sdk library
+        run: ./checkmac.sh && cargo build --package play-cpp-sdk --release --target x86_64-apple-darwin
+
+      - name: Build demo project
+        working-directory: demo
+        run: make x86_64_build_test
```

Comment:
Can this command run in `mac-build.yml`?

Reply:
yes, i'll move it